test/test_profiler.py

```diff
@@ -99,6 +100,21 @@ def forward(self, x):
 
         torch._C._set_graph_executor_optimize(prev_opt)
 
+class TestProfilerFlops(TestCase):
+    def test_function(self):
+        model = torch.nn.Sequential(
+            nn.Conv2d(1, 20, 5),
+            nn.ReLU(),
+            nn.Linear(1, 40),
+            nn.ReLU(),
+        )
+        inputs = torch.randn(20, 1, 5, 5)
+        with profiler.profile(record_shapes=True, with_flops=True) as prof:
+            with profiler.record_function("model_inference"):
+                model(inputs)
+        profiler_output = prof.key_averages(group_by_input_shape=True).table(sort_by="cpu_time_total", row_limit=10)
+        print(profiler_output)
+        self.assertIn("GLOPs", profiler_output)
```

Comment:
i think it'd be better if we supported automatic scaling,
that's check if we need to use TFLOPs, or GFLOPs in profiler
depending on the value

Reply:
GLOP -> GFLOPs